components/product/ProductCard/ProductCard.tsx

```diff
@@ -80,7 +80,7 @@ const ProductCard: FC<Props> = ({
             <div className={s.imageContainer}>
               <Image
                 quality="85"
-                src={src}
+                src={src || '/product-img-placeholder.svg'}
```

Comment:
@leahtard 
I would like to hear your motivation to import an SVG inline instead of importing it at the top and reusing it here and on line 60. Importing directly here, you may lose more performance, nothing significant, but it would be a request for each image not found. I would assume that there would always be no image on any of the items and import the placeholder at the top only once and use that reference for those that were not found. What do you think?

Reply:
Thank you for your review @marceloF5. I pushed up a small update that fixes the repeating image references. I did try to place the `svg` markup inline, but the Image component is type checking for an image reference.

Cheers, Leah

Reply:
Nice @leahtard! 🚀